client/pedersen.go

```diff
@@ -7,23 +7,48 @@ import (
 	"math/big"
 )
 
-func (c *Client) Pedersen(dlog *dlog.ZpDLog, val *big.Int) error {
-	c.handler.pedersenCommitter = commitments.NewPedersenCommitter(dlog)
+type pedersenCommonClient struct {
```

Comment:
Yes, definitely. Before refactoring we had a file `commitments.go` where we kept common functionality for both Pedersen and Pedersen_EC, but until now the common structs and functions were kept in `pedersen.go`. This is now fixed and we again have a seperate file for common functionality called `pedersen_common.go`